prerequisites/build-functions/build_and_install.sh

```diff
@@ -23,8 +23,8 @@ build_and_install()
 
     if [[ "${package_to_build}" == "mpich" && "${version_to_build}" == "3.2" ]]; then
       info "Patching MPICH 3.2 on Mac OS due to segfault bug (see http://lists.mpich.org/pipermail/discuss/2016-May/004764.html)."
-      sed -i '' 's/} MPID_Request ATTRIBUTE((__aligned__(32)));/} ATTRIBUTE((__aligned__(32))) MPID_Request;/g' \
-	  "${download_path}/${package_source_directory}/src/include/mpiimpl.h"
+      sed 's/} MPID_Request ATTRIBUTE((__aligned__(32)));/} ATTRIBUTE((__aligned__(32))) MPID_Request;/g' \
+        "${download_path}/${package_source_directory}/src/include/mpiimpl.h"
```

Comment:
@rouson Why did you do this? Now sed will just output the replacements to stdout rather than performing the patch.